run.py

```diff
@@ -15,6 +18,11 @@ def main():
 
     bot = Union(config=config)
 
+    try:
+        bot.load_extension("union.core.core")
+    except Exception as e:
+        print(f"Failed to load core cog.\n{e.__class__.__name__}: {e}")
```

Comment:
Per the discussion in the chat, we can leave this as-is for now and revisit it if we ever encounter a case where the core cog actually does fail to load for whatever reason.